### ISSUE TITLE
about jzy3d

### ISSUE THREAD

**Issue by AnaSafari** [retronym/jzy3d-demo#4]
Hello,
I understood I need to override the definition of methods and functions of jzy3d to make them valid to be used in scala. for example 
val m = new Mapper() { override def f(x:Double,y:Double): Double = x+y }
 
my question is, is there a way or an update that can help with not overriding the functions?

I would appreciate if you could help me?

**Comment by jzy3d** [comment]
No. But you could make some proposal that might be integrated to the api
